gammapy/maps/axes.py

```diff
@@ -1131,23 +1135,38 @@ def downsample(self, factor):
         if self.node_type == "edges":
             nbin = self.nbin / factor
 
-            if np.mod(nbin, 1) > 0:
-                raise ValueError(
-                    f"Number of {self.name} bins is not divisible by {factor}"
-                )
+            if strict and np.mod(nbin, 1) > 0:
+                idx = np.arange(0, self.nbin, factor)
+                if idx[-1] < self.nbin:
+                    idx = np.append(idx, self.nbin)
+                edges = self.edges[idx]
 
-            edges = self.edges[::factor]
-            return self.from_edges(edges, name=self.name, interp=self.interp)
+            else:
+                if np.mod(nbin, 1) > 0:
+                    raise ValueError(
+                        f"Number of {self.name} bins is not divisible by {factor}"
+                    )
+                edges = self.edges[::factor]
+            downsampled = self.from_edges(edges, name=self.name, interp=self.interp)
+            return downsampled
         else:
             nbin = (self.nbin - 1) / factor
 
-            if np.mod(nbin, 1) > 0:
-                raise ValueError(
-                    f"Number of {self.name} bins - 1 is not divisible by {factor}"
-                )
+            if strict and np.mod(nbin, 1) > 0:
```

Comment:
I don't think you need the separate handling for node type anymore either

Reply:
On second thoughts, I think you do. To be able to preserve the `node_type`
But then, this logic needs modification